src/main/java/Deadlines.java

```diff
@@ -0,0 +1,11 @@
+public class Deadlines extends Task {
+    String due_date;

```

Comment:
According to the coding standard, shouldn't due_date be dueDate?

Reply:
Alright noted. Thanks for the review!